<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>815</width>
    <height>589</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyAntivirus</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>421</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log Info</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Сканнер</string>
     </attribute>
     <widget class="QLabel" name="scannerLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сканнер</string>
      </property>
     </widget>
     <widget class="QLabel" name="pathLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к файлу</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>351</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="resultTextEdit">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>10</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startScanButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>260</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Мониторинг</string>
     </attribute>
     <widget class="QLabel" name="monitorLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Мониторинг</string>
      </property>
     </widget>
     <widget class="QLabel" name="pathLabel_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к файлу</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathLineEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="resultTextEdit_3">
      <property name="geometry">
       <rect>
        <x>398</x>
        <y>10</y>
        <width>371</width>
        <height>321</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="monitorButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Мониторинг</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelMonitorButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Сканнер с расписанием</string>
     </attribute>
     <widget class="QLabel" name="scheduleLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сканнер с расписанием</string>
      </property>
     </widget>
     <widget class="QLabel" name="pathLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к файлу</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathLineEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>321</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="resultTextEdit_2">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>17</y>
        <width>411</width>
        <height>231</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>127</width>
        <height>24</height>
       </rect>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="setButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Установить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelSchedule">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="scanStatusLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="monitorStatusLabel">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scheduleStatusLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>314</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
